help/blocks/columns/columns.css

```diff
@@ -1,27 +1,77 @@
-main .section .columns-wrapper .columns {
+.columns {
   background-color: var(--background-color);
+  font-size: var(--body-font-size-xs);
+  word-break: break-word;
+  margin-bottom: 64px;
 }
 
+.columns-wrapper:last-child .columns:last-child {
+  margin-bottom: 0;
+}
+
+/* article table - row */
 .columns > div {
+  flex-direction: unset;
   display: flex;
-  flex-direction: column;
+  gap: 12px;
+  padding: 24px 0;
+  border-bottom: 1px solid #ccc;
 }
 
+/* article table - cell / column */
 .columns > div > div {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+}
+
+/* article table - first row */
+.columns > div:first-child {
+  font-family: var(--th-font-family);
```

Comment:
this is basically a column now
can we have a meaningful variable name which doesn't include `th`

Reply:
changed